### PR TITLE
add config option for disabling bulk actions

### PR DESCRIPTION
**Pull request by rossedfort**
## What was changed

- Add a config option for disabling bulk actions in the UI
## Why?

- So that when we can release this feature we only need to change a config and not make code changes
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: config/development.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ workflowTerminateDisabled: false
 workflowCancelDisabled: false
 workflowSignalDisabled: false
 workflowResetDisabled: false
+bulkActionsDisabled: false
```
Comment (feedmeapples) [inline config/development.yaml:17]: @flossypurse @DJSanti do we communicate this to users as batch operations or bulk actions? In API we call this batch jobs / operations

Comment (rossedfort) [reply config/development.yaml:17]: @feedmeapples, @flossypurse and @DJSanti - some context. Internally we are differentiating bulk and batch as 2 different things. 

Bulk - triggering an action on a known set of `n` workflow executions where `n` could be anywhere from `1` to the `500` (depending on the page size the user has set)
Batch - triggering an action against _all_ workflow executions matching a query

The intent here is to mimic Gmail when your search matches more results than you can view on a "page"
Screen Shot 2023-01-31 at 3 57 04 PM

The purpose of this flag is to disable both, so I'm happy to rename it to `batchActionsDisabled` to align with other tools. Personally I'm against differentiating batch and bulk externally to users. I think of them as the same feature even though the internal implementation might be slightly different.

**Review by feedmeapples — APPROVED** [review]

File: config/development.yaml (hunk 1 above)
Comment (feedmeapples) [reply config/development.yaml:17]: thx for the context, bulk sounds good to me

Comment (rossedfort) [reply config/development.yaml:17]: @feedmeapples I actually prefer batch now that you've mentioned it, but just wanted to clarify that although we differentiate internally I think it's ok for this 1 config value to control both batch and bulk features in the UI 👍

**Pull request merged**